elastic/kibana
# [Security Solution] Merge risk input and asset integration expandable panels

## Summary

Update the user expandable flyout left panel to be consistent with Alerts flyout implementation. After this change, we always display the same left panel but dynamically create tabs for different sub-panels.

This issue can be considered a UI/UX bug because the user's experience of navigating between the tabs was poor and had some edge cases.

This PR also moves flyout files and folders to match the Alert flyout structure. The new folder structure looks like this:
```
public/flyout
  ...
  entity_details
    shared
    user_right
    user_details_left
      ...
      tabs
        ...
        asset_document.tsx
        risk_inputs.tsx
    
```


**No tab**
<img width="400px" src="https://github.com/elastic/kibana/assets/1490444/f0546410-7bd8-4685-9933-58879714a653">

**One tab**
<img width="600px" src="https://github.com/elastic/kibana/assets/1490444/7f9291bc-b617-4310-8bd1-acb77bec555e">


**Two tabs**
<img width="600px" src="https://github.com/elastic/kibana/assets/1490444/ba347124-ba71-4206-8453-ceff5455f156">

**Okta tab with table**
<img width="400px" src="https://github.com/elastic/kibana/assets/1490444/0cdae267-6be6-4e4f-bacd-5e7d2e778a3b">

**Okta tab with json**
<img width="400px" src="https://github.com/elastic/kibana/assets/1490444/e630581b-6122-4532-9f09-a7616b1fceac">


### How to test it
Please check the same section on this PR: https://github.com/elastic/kibana/pull/171629


### Checklist

Delete any items that are not applicable to this PR.
